### ISSUE TITLE
OServerAdmin credentials are over-written after a connection to a database is acquired through a connection pool factory

### ISSUE THREAD

**Issue by asthanasid** [orientechnologies/orientdb#5788]
```
OServerAdmin serverAdmin = new OServerAdmin("192.168.1.5").
                connect("root", "somerootpassword");
        OPartitionedDatabasePoolFactory factory = new OPartitionedDatabasePoolFactory(20);
        System.out.println("list" + serverAdmin.listDatabases());
        System.out.println("list" + serverAdmin.listDatabases());
        ODatabaseDocumentTx dbtx = factory.get("remote:192.168.1.5/www.orga.com",
                "asthana.siddhartha", "password").acquire();
        System.out.println("list" + serverAdmin.listDatabases());
```
First 2 calls to list databases go through fine. The last line throws exception:

```
com.orientechnologies.orient.core.exception.OSecurityAccessException: Server user not authenticated.

```

**Comment by tglman** [comment]
hi @asthanasid, 

which version of orientdb are you using ? i tried this on the last 2.1 and on 2.2-beta and it works correctly.

bye

**Comment by asthanasid** [comment]
Hi 

its oreintdb community edition 2.1.9

**Comment by tglman** [comment]
Hi @asthanasid,

I think this was fixed in a recent version of 2.1.x so i'll suggest you to update on the last hotfix of 2.1 (currently 2.1.16)
